src/main.js

```diff
@@ -2,5 +2,8 @@ import { createApp } from 'vue';
 import App from './App.vue';
 import router from './route';
 import './index.css';
+import axios from 'axios';
+
+axios.defaults.baseURL = 'http://127.0.0.1:8000/api';
```

Comment:
That's great, I'm ok with it.
But you can take a look at this to use an .env file to store this kind of information :) https://vitejs.dev/guide/env-and-mode.html